docs/detections/rules-ui-manage.asciidoc

```diff
@@ -131,17 +131,15 @@ You cannot export Elastic prebuilt rules, but you can duplicate a prebuilt rule,
 If you try to export with both prebuilt and custom rules selected, only the custom rules are exported.
 ====
 
-The following configuration items are also included in the `.ndjson` file:
+The `.ndjson` file also includes any actions and exception lists related to the exported rules. However, other configuration items require additional handling when exporting and importing rules:
 
-* Actions
-* Exception lists
+- *Data views*: For rules that use a {kib} data view as a data source, the exported file contains the associated `data_view_id`, but does _not_ include any other data view configuration. To export/import between {kib} spaces, first use the {kibana-ref}/managing-saved-objects.html#managing-saved-objects-share-to-space[Saved Objects] UI (*Stack Management* -> *Kibana* -> *Saved Objects*) to share the data view with the destination space.
```

Comment:
```suggestion
- *Data views*: For rules that use a {kib} data view as a data source, the exported file contains the associated `data_view_id`, but does _not_ include any other data view configuration. Use {kib}'s {kibana-ref}/managing-saved-objects.html#managing-saved-objects-export-objects[Saved Objects] UI (*Stack Management* -> *Kibana* -> *Saved Objects*) to export and import any necessary data views _before_ you export and import the detection rules.
```

Reply:
Only recommending this because I liked the structure you used for describing steps to export/import connectors through saved objects. I'm not sure why GH is italicizing the revised version. Guess it's a lil funky today.

Reply:
@nastasha-solomon I think this loses some of the original meaning, which was about exporting/importing rules _across spaces_ in the same cluster. And the user doesn't use the Saved Objects UI to export/import the data view, only to share the data view with another space. But I'm not very familiar with this, so pinging @yctercero for the tech perspective.